servatrice/check_schema_version.sh

```diff
@@ -31,3 +31,5 @@ if ! grep -q "$expected_define" servatrice/src/servatrice_database_interface.h;
     echo "servatrice_database_interface.h does not contain expected #define: $expected_define"
     exit 1
 fi
+
+set +e
```

Comment:
This doesn't make sense at all. There's no statements which follow this, so I don't see how this can have any effect. Can you elaborate on what this fixes specifically? If you could point me to any bash documentation that shows how this would affect things that would help too.

I guess we'll also see once the travis builds finish